### DISCUSSION TITLE
How to have multiple MPV windows open?

### DISCUSSION THREAD

**Discussion by magnilens** [tomasklaen/uosc#693] [Q&A]
Occasionally I may want to open a second MPV window, either to compare with the first video or check on something else, without closing the first window. Is there a way to configure MPV/uosc to allow this?

**Comment by christoph-heinrich** [comment]
Do you mean something like `run mpv ${path}` ?

**Comment by magnilens** [comment]
Nevermind. I just discovered that MPV already works that way. Strange, I got the impression that it didn't a few weeks ago. Might have been mpv.net instead?
